lib/publiccloud/utils.pm

```diff
@@ -49,6 +49,52 @@ sub select_host_console {
     set_var('TUNNELED', $tunneled) if $tunneled;
 }
 
+sub register_addon {
+    my ($remote, $addon) = @_;
+    my $regcode = get_required_var('SCC_REGCODE');
```

Comment:
This is never used within this subroutine

Reply:
line 60

Reply:
But in line 59 it is overwritten with `SCC_REGCODE_LTSS`, so this var is never used.

Reply:
done